sources/osgWrappers/serializers/osg.js

```diff
@@ -1,7 +1,16 @@
 'use strict';
 var P = require('bluebird');
+var Notify = require('osg/notify');
```

Comment:
If I recall @cedricpinson prefer notify with no capital letter at the beginning from other PR's.

Reply:
About where to put the rejectObject I'm also not clear, but if you don't find a better place maybe you can add it to the osg/Utils hodgepodge.